services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js

```diff
@@ -196,13 +196,17 @@ let EditNoticeOfDepartureForm = (props) => {
           </Col>
           <Col span={3}>
             <p className="field-title">&nbsp;</p>
-            <LinkButton
-              className="nod-table-link"
-              onClick={() => downloadFileFromDocumentManager(checklist)}
-              title={checklist?.document_name}
-            >
-              Download
-            </LinkButton>
+            {checklist?.document_name ? (
```

Comment:
What about other files. We are not listing it hee?

Reply:
For the other files, the section showing them only shows if any files exist, so it won't have this problem